.github/workflows/lint.yml

```diff
@@ -1,8 +1,11 @@
 name: lint
 
 on:
-  push:
+  workflow_call:
   workflow_dispatch:
+  pull_request:
+    branches: [main]
+  push:
```

Comment:
Is this going to double run for PR/push?

Reply:
yes, for linting it will double run